src/automations/Job.ts

```diff
@@ -254,9 +254,10 @@ export default class Job {
     private async loadAllJobs() {
         await this.page.waitForSelector(".job");
         let morePageButton = await this.page.$("#show_more_jobs");
+
         while (morePageButton) {
             morePageButton.click();
-            await this.page.waitForNetworkIdle();
+            await this.page.waitForTimeout(3000);
```

Comment:
is there a scenario where you can have 10 pages?

Reply:
I dont know

Reply:
how many pages do you see as an admin?

Reply:
6 pages

Reply:
`waitForTimeout` it a very risky approach were if someones internet or PC is slow the result may differ, there are multiple ways to wait in puppeteer, if `waitForNetworkIdle` isn't working try providing an argument, see [this comment](https://github.com/puppeteer/puppeteer/issues/1552#issuecomment-350954419) for more info about the options.
Otherwise you can use [`waitForSelector`](https://puppeteer.github.io/puppeteer/docs/puppeteer.page.waitforselector/) if you are waiting for a given UI element.
There different `waitFor` methods that puppeteer provides, the last one that you will need is the hard written delay one!

Reply:
there is no element to check i will check the job count